lib/dom.js

```diff
@@ -831,7 +863,6 @@ Document.prototype = {
 		node.nodeName = tagName;
 		node.tagName = tagName;
 		node.localName = tagName;
-		node.childNodes = new NodeList();
 		var attrs	= node.attributes = new NamedNodeMap();
 		attrs._ownerElement = node;
```

Comment:
... including moving this to the Element constructor?
(The same is true for `createElementNS`)